code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -146,10 +146,15 @@
 	X.create_shriekwave() //Adds the visual effect. Wom wom wom
 	//stop_momentum(charge_dir) //Screech kills a charge
 
-	for(var/mob/living/L in range(world.view, X))
+	var/screech_range = world.view

```

Comment:
If you're using this just in the proc then there's zero point to having this instead of world.view. It doesn't allow customization and only further obfuscates the code.